lib/srv/uacc/uacc_linux.go

```diff
@@ -0,0 +1,196 @@
+// +build linux
+
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package uacc concerns itself with updating the user account database and log on nodes
+that a client connects to with an interactive session.
+*/
+package uacc
```

Comment:
I'm curious why you named this package `uacc`? In the comment you mention this is for updating the user account database and all the linux code is focused on `utmp`.

What does `uacc` mean?

Reply:
User accounting. If we were to implement OSX support later, it would go in a similar implementation file in the same package.